home work 20/script.js

```diff
@@ -0,0 +1,13 @@
+function removedrinks() {
+    let drinksList = document.querySelector('#drinks-list');
+    let drinksArray = Array.from(drinksList.children).map((li) => li.textContent);
+    drinksList.innerHTML = '';
+    let uniqueDrinks = [...new Set(drinksArray)].sort().forEach((drink) => {
+        let li = document.createElement('li');
+        li.textContent = drink;
+        drinksList.appendChild(li);
+    });
+}
+removedrinks();
+
```

Comment:
Good. But:
1. bad name for function! Let's think of a name. Function removes duplicate drinks and sorts them. So maybe it's better to call `removeDuplicatesAndSort()`.
2.  Variable `uniqueDrinks` is useless
3. Let me show you how to do it differently. (although your version is not bad either)
```javascript
function removeDuplicatesAndSort() {
    let drinksList = document.querySelector('#drinks-list');
    let drinksArray = Array.from(drinksList.children).map((li) => li.textContent);
    let uniqueDrinks = Array.from(new Set(drinksArray));
  
    drinksList.innerHTML = '';
    uniqueDrinks.sort().forEach((drink) => {
        drinksList.innerHTML += `<li>${drink}</li>`;
    });
}
removeDuplicatesAndSort()
```